exercise/templates/exercise/_user_results.html

```diff
@@ -162,6 +162,13 @@ <h3 class="panel-title">
 						<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
 						{% translate "VIEW_SUBMISSIONS" %}
 					</a>
+					{% if entry.submissions %}
+					{% url 'submitter-inspect' course_slug=course.url instance_slug=instance.url module_slug=module.url exercise_path=entry.get_path user_id=student.id as inspect_url %}
```

Comment:
This `_user_results.html` template is also used in the "Your points" page. This inspect link is broken there because the template context does not include the `student` variable. At any rate, the page shows the teacher's own points for teachers (and for students, her/his own points), thus the inspect link is really not useful under "Your points". The teacher does not need a shortcut link for inspecting her/his own submission. The inspect link is not visible to students (since the inspect view is only for teachers).

I think this new inspect link must be removed from the "Your points" page. Perhaps the easiest way to implement that is to change the if condition to `if entry.submissions and student.id`.